pkg/utils/utils.go

```diff
@@ -287,10 +289,26 @@ func setNetdevMTU(pciAddr string, mtu int) error {
 }
 
 func tryGetInterfaceName(pciAddr string) string {
-	names, err := dputils.GetNetNames(pciAddr)
-	if err != nil || len(names) < 1 {
+	var names []string
```

Comment:
Yes same on my side both intel and mellanox so I add this small wait.

is not critical because the refresh will get the data but I want it to be more stable

Reply:
When there are a lot of VFs with vfio-pci driver, thus no interface name. The total waiting time could be huge. And it will occur in every refresh cycle. So I prefer we keep it as it is.